templates/publisher/publicise/embedded_cards.html

```diff
@@ -57,18 +57,13 @@ <h4>Promote your snap using embeddable responsive card</h4>
       <label>HTML:</label>
     </div>
     <div class="col-7">
-      <div class="p-code-copyable not-cli">
-        <code class="p-code-copyable__input" id="snippet-card-html">&lt;iframe src="https://snapcraft.io/{{ snap_name }}/embedded?button=black&channels=true&summary=true&screenshot=true" frameborder="0" width="100%" height="320px" style="border: 1px solid #CCC; border-radius: 2px;" &gt;&lt;/iframe&gt;</code>
-        <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-card-html">Copy to clipboard</button>
+      <div class="p-code-snippet">
+        <pre class="p-code-snippet__block is-wrapped" style="white-space: pre-wrap;"><code id="snippet-card-html">&lt;iframe src="https://snapcraft.io/{{ snap_name }}/embedded?button=black&channels=true&summary=true&screenshot=true" frameborder="0" width="100%" height="320px" style="border: 1px solid #CCC; border-radius: 2px;"&gt;&lt;/iframe&gt;</code></pre>
```

Comment:
Same as above - avoid adding custom styles inline, add them in SCSS so they can be removed when Vanilla is released.

```suggestion
        <pre class="p-code-snippet__block is-wrapped"><code id="snippet-card-html">&lt;iframe src="https://snapcraft.io/{{ snap_name }}/embedded?button=black&channels=true&summary=true&screenshot=true" frameborder="0" width="100%" height="320px" style="border: 1px solid #CCC; border-radius: 2px;"&gt;&lt;/iframe&gt;</code></pre>
```